modin/config/envvars.py

```diff
@@ -642,6 +642,13 @@ class ExperimentalNumPyAPI(EnvironmentVariable, type=bool):
     default = False
 
 
+class ExperimentalGroupbyImpl(EnvironmentVariable, type=bool):
+    """Set to true to use Modin's experimental group by implementation."""
```

Comment:
I think we should put some more clarification regarding what the exp impl implies.

Reply:
done